tests/test_pysigma_integration.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from sigma.plugins import InstalledSigmaPlugins
+from sigma.backends.loki import LogQLBackend
+from sigma.pipelines.loki import (
+    loki_grafana_logfmt,
+    loki_promtail_sysmon,
+    loki_okta_system_log,
+)
+
+
+@pytest.fixture
+def installed() -> InstalledSigmaPlugins:
+    return InstalledSigmaPlugins.autodiscover()
+
+
+def test_auto_discover_loki_be(installed: InstalledSigmaPlugins):
+    assert "loki" in installed.backends
+    assert installed.backends["loki"] is LogQLBackend
```

Comment:
Better to use `isinstance` for assertion of type here:
```suggestion
    assert isinstance(installed.backends["loki"], LogQLBackend)
```

Reply:
I don't think it is better to use `isinstance` here. `installed.backends["loki"]` doesn't contain an instance of the LogQLBackend - it contains the class - and `isinstance(installed.backends["loki"], LogQLBackend)` correctly returns false.